lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -78,12 +78,14 @@
                     </mat-menu>
                 </ng-container>
 
-                @if (showToolbarDividers) {
```

Comment:
You're deleting existing feature that allows to hide each dividers, this will break the upstream in other repos